README.md

```diff
@@ -36,10 +36,17 @@ import alias from 'rollup-plugin-alias';
 export default {
   input: './src/index.js',
   plugins: [alias({
-    somelibrary: './mylocallibrary'
+    resolve: ['.jsx', '.js'],
+    entries:[
+      {find:'somelibrary', replacement: './mylocallibrary'},
+      {find:^./foobar\/path.*/i, replacement: './bar', isRegEx:true} 
+    ]
   })],
 };
 ```
+The order of the entries are important, in that are the first rules are applied first (obviously). :) 
```

Comment:
`The order of the entries is important, in that the first rules are applied first `

Reply:
Fixed